assets/terraform-modules/node/templates/node.yaml.tmpl

```diff
@@ -152,6 +152,18 @@ storage:
           [Resolve]
           DNS=${host_dns_ip}
           Domains=~.
+    - path: /etc/docker/daemon.json
```

Comment:
This file gets lengthy, I think at some point we should split it into separate files as well. Like: `users`, `docker`, `kubelet`.

Created an issue for it #1376.

Reply:
Yes, I guess so :)